### ISSUE TITLE
Deezer local plugin difficult to install under current HA LMS

### ISSUE THREAD

**Issue by anotherstuart** [pssc/ha-addon-lms#72]
Hi

You may be aware of the work to create a local Deezer plugin since the mysqueezebox doesn't support it any more.

I know you also are aware that v8.5.0 of LMS will take out the plugins that don't work anymore.

The current issue is that the new Deezer (local) plugin does not install properly with the current HA version of LMS and I wondered if there could be a quick fix, pre v8.5, to remove the old Deezer plugin from the HA version, pending the full 8.5 release in HA.

(The new Deezer local plugin can be installed using a zip and a lot of fiddling but it is not very easy and certainly not any good for regular updates as the plugin is being developed to test it)

Or is that all too unofficial??

Thanks

Stuart

**Comment by pssc** [comment]
Is this current plugin working?

You should just able able to run an init script to delete the old plugin...

Sure we can work something out

**Comment by anotherstuart** [comment]
Hi

Yes, I have got the plugin working. It is great.

To install it (I am not really sure how I managed to) I had to fiddle around, 'wget' the zip file into the plugin folder that I found then unzip inside a folder and I think I mv'd it having renamed the existing 'Deezer' folder.  I I assume it overwrote the built in Deezer plugin. But I had to try several times and not sure what made it come up properly in the end.

It is now working fine (in fact, it's amazing!) and I daren't upgrade to the more recent releases of the plugin for fear I never get it installed in the right place next time!

Thanks

Stuart
